l10n_ch_payment_register/wizard/payment_order_create.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (c) 2015 brain-tec AG (http://www.braintec-group.com)
+#    All Right Reserved
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import time
+from lxml import etree
+
+from openerp import models, fields, api, _
+
+
+class PaymentOrderCreate(models.TransientModel):
+    """
+    Create a payment object with lines corresponding to the account move line
+    to pay according to the date and the mode provided by the user.
+    Hypothesis:
+    - Small number of non-reconciled move line, payment mode and
+    bank account type,
+    - Big number of partner and bank account.
+
+    If a type is given, unsuitable account Entry lines are ignored.
+    """
+
+    _name = 'payment.order.create'
+    _description = 'payment.order.create'
+
+    duedate = fields.Date('Due Date', required=True,
+                          default=lambda *a: time.strftime('%Y-%m-%d'))
+
+    entries = fields.Many2many('account.move.line', 'line_pay_rel', 'pay_id',
+                               'line_id')
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
+                        submenu=False):
+        res = super(PaymentOrderCreate, self).\
+            fields_view_get(view_id=view_id, view_type=view_type,
+                            toolbar=toolbar, submenu=False)
+
+        context = dict(self._context or {})
```

Comment:
self._context will always be a frozen dict, no need for the check and to copy it